package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "restapp-boilerplate",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
Is this really a major version? There shouldn't be something that breaks backwards compability? Maybe the change from using `bind` on the controller and instead using function expressions?

Reply:
@alexandernanberg What do you think?